### ISSUE TITLE
PHP 7.4.9 - HTTP-Error 500.0 - Internal Server Error

### ISSUE THREAD

**Issue by amelsens** [nawawi/docket-cache#4]
Hello,
I use PHP 7.4.9 with WordPress 5.5 and your latest version of Docket Cache plugin and I get now the most of all the page requests, the follow error: HTTP-error 500.0 – Internal Server Error. C:\PHP\v7.4.9\php-cgi.exe - The FastCGI-proces is unexpectedly terminated 
When I go back to PHP version 7.4.8 I don’t have any problems.
Maybe it’s something to do with the Opcache extension of PHP 7.4.9?
Or maybe the following Opcache settings in my php.ini file are not longer good with the new version of PHP 7.4.9?
The Opcache settings I use are:
[PHP_OPCACHE]
extension=php_opcache.dll
zend_extension=php_opcache.dll
opcache.enable=1
opcache.memory_consumption=512
opcache.interned_strings_buffer=8
opcache.max_accelerated_files=50000
opcache.use_cwd=1
opcache.validate_timestamps=1
opcache.enable_file_override=0
opcache.dups_fix=0
opcache.revalidate_freq=2
opcache.consistency_checks=0
opcache.max_wasted_percentage=5
opcache.revalidate_freq=2
opcache.enable_cli=0
opcache.file_cache=”C:\Windows\Temp\”
opcache.file_cache_consistency_checks=1
opcache.file_cache_fallback=1

Please, can this be solved? Or what are the best Opcache settings in php.ini?
I can set everything myself. I use a localhost Windows IIS webserver environment.
Sorry, I don’t have an online URL.

I wrote the same message on the Support forum on www.wordpress.org.

Thanks in advance.
Kind Regards.

**Comment by nawawi** [comment]
Hi,

1) Enable PHP 7.4.8
2) Deactivate and uninstall docket cache
3) Manual delete the cache folder wp-content/cache/docket-cache and wp-content/object-cache.php file if still exists
4) Enable PHP 7.4.9
5) Install fresh docket cache from wordpress
6) Activate and see if have same error

can you provide php error log?

Thanks.

**Comment by amelsens** [comment]
Hi,
First of all, thanks for you fast a clear support. :)
In fact you're right, the "Visual C++ Redistributables" must be always the latest version as the latest version of PHP. So, I have installed the latest versions for Visual C++ of Visual Studio 2019.
Then I have followed your great instructions to solve the problem.
And in fact, no longer the same HTTP-Error 500.0 anymore! Everything is working as before. So, I have replaced my previous Opcache settings and evrything works as expected.
Really, what a fast and very clear and usefull explantion to resolve the problem. As simple as that....
Thank you very much! :)
Kind Regards.
